devel/swiftlint/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        realm swiftlint 0.52.4
+github.setup        realm swiftlint 0.56.2
```

Comment:
latest upstream version is `0.57.0` - has that resolved some of the build issues?